modules/backend/widgets/Lists.php

```diff
@@ -1685,6 +1685,7 @@ public function onApplySetup()
      */
     public function onResetSetup()
     {
+        $this->putSession('order', null);
```

Comment:
Can we just make the order a userPreference same as the other ones? Any reason why this is managed a different way than the other ones?

Reply:
No idea why, I can convert to a user preference if you want.